CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 * [ENHANCEMENT] Store-gateway: improved performance of series matching. #3391
 * [ENHANCEMENT] Move the validation of incoming series before the distributor's forwarding functionality, so that we don't forward invalid series. #3386 #3458
 * [ENHANCEMENT] S3 bucket configuration now validates that the endpoint does not have the bucket name prefix. #3414
+* [ENHANCEMENT] Memcached: added extra `Del`method for cache item invalidation. #3515
```

Comment:
We're used to mention in the CHANGELOG only user-facing changes. This change is not user-facing, so I would remove it.